Implement a new way of displaying inbound references in the Delete di…

ashklianko opened the pull request:
…alog #3260

ashklianko committed:
Implement a new way of displaying inbound references in the Delete dialog #3260

ashklianko committed:
Implement a new way of displaying inbound references in the Delete dialog #3260

-codacy

alansemenov commented on modules/lib/src/main/resources/i18n/dialogs.properties:
```diff
@@ -109,6 +106,9 @@ dialog.rename.label=New name
 dialog.confirm.resetInheritance=Resetting this content item will revert all the local changes. This action cannot be undone.
 dialog.project.delete.confirm.subheader=You are about to delete this project with all of its content. This action cannot be undone.
 dialog.layers.notAvailable=No access to the project
+dialog.delete.hasInbound.part1=Inbound dependencies!
```
Can you rename `dialog.delete.dependencies` to `dialog.delete.hasInbound.part1` (and add "!") in all other language files as well, so that we could reuse the existing translations of this phrase?

alansemenov reviewed (changes requested):
One more thing (that should've been mentioned in the original task description): we should change the order of child items so that the ones with the inbound references are shown on top of the list (just like we do with invalid/in progress items in the Publish dialog).

alansemenov commented:
There's also some Codacy issues:

![image](https://user-images.githubusercontent.com/11612490/124462244-cd579e80-dd91-11eb-8092-5cd6c8b6f9a6.png)

ashklianko committed:
Merge remote-tracking branch 'remotes/origin/master' into issue-3260

ashklianko committed:
Implement a new way of displaying inbound references in the Delete dialog #3260

-showing items with inbound refs first in the list of dependant items
-Updated lang entries
-Codacy fix in .less files

ashklianko commented:
Updated

alansemenov reviewed (approved):
One more thing, that should've been mentioned in the original task description
